src/hooks/VideoPlayer.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+
+import { VideoContext } from '../shared/VideoContext'
+
+export const VideoPlayer = ({
+  autoplay = false,
+  children,
+  startAt = 0,
+  style,
+}) => {
+  const video = React.useRef()
+  const [isPlaying, setIsPlaying] = React.useState()
+  const pause = () => {
+    setIsPlaying(false)
+    video.current.pause()
+  }
+  const play = () => {
+    setIsPlaying(true)
+    video.current.play().catch(() => false)
+  }
+  const setVideoRef = ref => {
+    video.current = ref
+
+    if (autoplay && video.current) {
```

Comment:
Can `video.current` be `false`? I don't know a ton about the ref callbacks, but can they be called with `undefined` and such?

Reply:
initially `video.current` will be undefined until it is set.

Reply:
but wasn't it just set two lines above?